### ISSUE TITLE
Inquiry regarding effective learning rate/batch size for full ImageNet dataset

### ISSUE THREAD

**Issue by jiseokkwak** [serendipity1122/Pre-trained-Model-Guided-Fine-Tuning-for-Zero-Shot-Adversarial-Robustness#3]
Hi, 

I'm now reproducing your work with ImageNet2012 dataset..

Do you know the best configuration for training on ImageNet2012 with 128-batch size?

I guess {learning_rate, weight_decay} = {1e-5, 1e-4} which is widely used for ViT-B/32 finetuning with batch size of 128 can be 
a good choice.

Do you have any idea??

Thank you
